File: src/main/java/Show_Activity.java
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.actionSystem.ToggleAction;
import org.jetbrains.annotations.NotNull;

public class Show_Activity extends ToggleAction {

    boolean selection = false;

    @Override
    public boolean isSelected(@NotNull AnActionEvent anActionEvent) {
        return selection;
    }

    @Override
    public void setSelected(@NotNull AnActionEvent anActionEvent, boolean b) {
        selection = !selection;
    }
}

File: src/main/kotlin/Actions/Show_Activity.kt
package Actions

import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.actionSystem.ToggleAction

class Show_Activity : ToggleAction() {
    var selection = false
    override fun isSelected(anActionEvent: AnActionEvent): Boolean {
        return selection
    }

    override fun setSelected(anActionEvent: AnActionEvent, b: Boolean) {
        selection = !selection
    }
}